code/modules/hydroponics/seed_datums.dm

```diff
@@ -540,7 +540,17 @@ var/global/list/gene_tag_masks = list()   // Gene obfuscation for delicious tria
 
 //Harvest without concern for the user
 /datum/seed/proc/autoharvest(var/turf/T, var/yield_mod = 1)
-	if(T && (!isnull(products)) && products.len && (yield > 0))
+	if(!T)
+		return
+	if(!products.len)
+		return
+	var/count = 0
+	for(var/obj/O in T)
```

Comment:
You can still move the objects, maybe with a conveyor, and still create an unlimited amount of items.

Reply:
Sounds like an edge case, I don't think we need to worry about that too much